compact.cpp

```diff
@@ -32,7 +32,7 @@ unsigned __stdcall ObjectMemory::lastOTEntry()
 	const OTE* pOT = m_pOT;
 	while (pOT[i].isFree())
 	{
-		ASSERT(i >= 0);
+		ASSERT(0 < i);
```

Comment:
Yes, there are merge conflicts. I've done the merge and am running tests before I do the commit.